core/src/wallet/currencies.cpp

```diff
@@ -228,6 +229,38 @@ namespace ledger {
                             .unit("satoshi", 0, "satoshi")
                             .unit("decred", 8, "DCR")
                             .unit("milli-decred", 5, "mDCR");
+            const api::Currency ETHEREUM =
+                    Currency("ethereum")
+                            .bip44(60)
+                            .forkOfEthereum(networks::getEthLikeNetworkParameters("ethereum"))
+                            .paymentUri("ethereum")
+                            .unit("wei", 0, "wei")
+                            .unit("ether", 18, "ETH")
+                            .unit("kwei", 3, "kwei")
+                            .unit("mwei", 6, "mwei")
+                            .unit("gwei", 9, "gwei");
+
+            const api::Currency ETHEREUM_ROPSTEN =
+                    Currency("ethereum_ropsten")
+                            .bip44(1)
+                            .forkOfEthereum(networks::getEthLikeNetworkParameters("ethereum_ropsten"))
+                            .paymentUri("ethereum")
+                            .unit("wei", 0, "wei")
+                            .unit("ether", 18, "ETH")
+                            .unit("kwei", 3, "kwei")
+                            .unit("mwei", 6, "mwei")
+                            .unit("gwei", 9, "gwei");
+
+            const api::Currency ETHEREUM_CLASSIC =
+                    Currency("ethereum_classic")
+                            .bip44(1)
```

Comment:
Is that `1` for Testnet?

Reply:
Yes it should be 1, did you find another source saying something different ?

Reply:
I was just confused by the name that doesn’t seem to be test ^^